package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngxf",
-  "version": "9.0.0",
+  "version": "9.0.1",
```

Comment:
I think that is an accident
```suggestion
  "version": "9.0.0",
```

Reply:
i just ran `npm install`, this is the version from the `package.json`